airtime_mvc/application/Bootstrap.php

```diff
@@ -36,10 +36,15 @@
 Logging::setLogPath('/var/log/airtime/zendphp.log');
 
 // We need to manually route because we can't load Zend without the database being initialized first.
-if (array_key_exists("REQUEST_URI", $_SERVER) && (strpos($_SERVER["REQUEST_URI"], "/provisioning/create") !== false)) {
+// We should probably look for a better way to do this rather tan overloading this if statement
+if (array_key_exists("REQUEST_URI", $_SERVER) && (stripos($_SERVER["REQUEST_URI"], "/provisioning/create") !== false)) {
     $provisioningHelper = new ProvisioningHelper($CC_CONFIG["apiKey"][0]);
     $provisioningHelper->createAction();
     die();
+} else if (array_key_exists("REQUEST_URI", $_SERVER) && (stripos($_SERVER["REQUEST_URI"], "/provisioning/change") !== false)) {
```

Comment:
The /provisioning/change can just be a regular controller action. The station should be configured with a working airtime.conf before we need to call /provisioning/change... (I think?) We can discuss this...